distribution/galleon-feature-packs/server-galleon-pack/src/main/resources/feature_groups/infinispan-dist-web.xml

```diff
@@ -30,9 +33,15 @@
                 <feature spec="subsystem.infinispan.cache-container.replicated-cache.component.transaction">
                     <param name="mode" value="BATCH"/>
                 </feature>
+                <feature spec="subsystem.infinispan.cache-container.replicated-cache.component.expiration">
+                    <param name="lifespan" value="900000000000000000"/>
+                </feature>
             </feature>
-            <feature spec="subsystem.infinispan.cache-container.distributed-cache">
-                <param name="distributed-cache" value="routing"/>
+            <feature spec="subsystem.infinispan.cache-container.replicated-cache">
```

Comment:
By default for WF, the `routing` cache for `infinispan-dist-web` feature group has `replicated` type instead of `distributed`.